pkg/worker/workerhandler/claimcleanuphandler/intern_ensure.go

```diff
@@ -0,0 +1,83 @@
+package claimcleanuphandler
+
+import (
+	"time"
+
+	"github.com/uvio-network/apiserver/pkg/object/objectlabel"
+	"github.com/uvio-network/apiserver/pkg/storage/poststorage"
+	"github.com/uvio-network/apiserver/pkg/storage/votestorage"
+	"github.com/xh3b4sd/objectid"
+	"github.com/xh3b4sd/rescue/task"
+	"github.com/xh3b4sd/tracer"
+)
+
+const (
+	oneDay = 24 * time.Hour * 7 // TODO revert to one day once the worker ran on testnet
+)
+
+func (h *InternHandler) Ensure(tas *task.Task) error {
+	var err error
+
+	var now time.Time
+	var day time.Time
+	{
+		now = time.Now().UTC()
+		day = now.Add(-oneDay)
+	}
+
+	// Look for all claims of lifecycle phase propose using microsecond scores. We
+	// have to use microseconds because the underlying claims are indexed using
+	// their IDs as scores, where those IDs are in randomized microsecond format
+	// based on claim creation time.
+	var pos poststorage.Slicer
+	{
+		pos, err = h.sto.Post().SearchTime(objectlabel.LifecyclePropose, day.UnixMicro(), now.UnixMicro())
+		if err != nil {
+			return tracer.Mask(err)
+		}
+	}
+
+	{
+		pos = pos.ObjectPending()
+	}
```

Comment:
We still need to remove those claims that have been created within the past 24 hours. Only pending claims older than one day should be purged.